paddle/fluid/memory/detail/memory_block.cc

```diff
@@ -38,7 +39,10 @@ MemoryBlock* MemoryBlock::GetRightBuddy(MetadataCache* cache) {
 void MemoryBlock::Split(MetadataCache* cache, size_t size) {
   auto desc = cache->LoadDesc(this);
   // make sure the split fits
-  PADDLE_ENFORCE_GE(desc->total_size, size);
+  PADDLE_ENFORCE_GE(desc->total_size, size,
+                    platform::errors::InvalidArgument(
+                        "The size of memory block to split is "
```

Comment:
`The size of memory (%d) to free exceeds the size of allocated cuda pinned memory (%d)`, is it possible to use this form to inform the specific size?